insights/parsers/dmsetup.py

```diff
@@ -9,6 +9,9 @@
 DmsetupInfo - command ``dmsetup info -C``
 -----------------------------------------
 
+DmsetupStatus - command ``dmsetup status``
+------------------------------------------
```

Comment:
This is not good adding 2 titles for 1 module, please just leave one which is general for this module and make the current 2 as sub titles.

Reply:
My bad, I forgot to unfold the closet code above